EventFilter/EcalRawToDigi/python/ecalDigis_cff.py

```diff
@@ -5,3 +5,24 @@
 ecalDigis = _ecalEBunpacker.clone()
 
 ecalDigisTask = cms.Task(ecalDigis)
+
+# process modifier to run on GPUs
+from Configuration.ProcessModifiers.gpu_cff import gpu
+
+# GPU-friendly EventSetup modules
+from EventFilter.EcalRawToDigi.ecalElectronicsMappingGPUESProducer_cfi import ecalElectronicsMappingGPUESProducer
+
+# raw to digi on GPUs
+from EventFilter.EcalRawToDigi.ecalRawToDigiGPU_cfi import ecalRawToDigiGPU as _ecalRawToDigiGPU
+ecalDigisGPU = _ecalRawToDigiGPU.clone()
+
+# copy the digi from the GPU to the CPU and convert to legacy format
+from EventFilter.EcalRawToDigi.ecalCPUDigisProducer_cfi import ecalCPUDigisProducer as _ecalCPUDigisProducer
+_gpu_ecalDigis = _ecalCPUDigisProducer.clone(
+  digisInLabelEB = 'ecalDigisGPU:ebDigisGPU',
+  digisInLabelEE = 'ecalDigisGPU:eeDigisGPU',
+  produceDummyIntegrityCollections = True,
+)
+gpu.toReplaceWith(ecalDigis, _gpu_ecalDigis)
```

Comment:
This could be
```python
gpu.toReplaceWith(ecalDigis, _ecalCPUDigisProducer.clone(
￼  digisInLabelEB = 'ecalDigisGPU:ebDigisGPU',
￼  digisInLabelEE = 'ecalDigisGPU:eeDigisGPU',
￼  produceDummyIntegrityCollections = True,
￼))
```

Reply:
Yes - I was thinking that will want both the fixed-`gpu` modifier version, and the `SwitchProducer` version, so `_gpu_ecalDigis` would be used for both.